test.js

```diff
@@ -504,8 +513,8 @@ function runTest (client, db) {
     })
   })
 
-  test('look up for expired packets', function (t) {
-    t.plan(7)
+  test.only('look up for expired packets', function (t) {
```

Comment:
this will run only this test, you need to remove `only`:

```suggestion
  test('look up for expired packets', function (t) {
```

Reply:
Ops, sorry missed that